docs/02-about-us/culture.md

```diff
@@ -32,6 +32,23 @@ We invest extra effort into our communication strategies because we are a mostly
 * **Balance Scores** - At every meeting, we each report our “balance score” -- a number from 1 to 10 that represents how well you are recognizing and honoring your priorities in your personal, work, and spiritual life. Everyone knows about the struggle for “work-life balance”, and this practice is our way of empowering people to honor theirs -- with the addition of spiritual/mental health as well. A high balance score doesn’t necessarily mean everything is going perfectly in your life, but it means you are honoring the priorities you have set for yourself. By hearing the balance of other team members, we can remain attuned to who might need extra support, or who is thriving and might have capacity to serve as a resource for others.
 * **Culture Videos** - Captured at the 2016 retreat, we have [several videos](https://vimeo.com/civicactions) that feature team members talking about CivicActions. It’s an interesting glimpse into the different values and appreciations from our peers. We invest a lot of energy into appreciating each other and creating a company where people are free to be themselves and grow their skills by taking risks and learning from failure.
 
+##Transparency
+
+**What is Radical Transparency?**
+
+We talk a lot about radical transparency here at CivicActions, but what is it?  There are many different, often conflicting definitions for transparency in business.  Here are some, from the book Accountability and Transparency for Peaceful Development by Kelly Ngyah.  You can read the relevant sections for free here. 
+
+*“[T]ransparency means operating in such a way that others can actually see how, what and why actions are carried out.”  As a principle...managers and directors of companies and organizations and board trustees have a duty to act visibly, predictably and understandably in order to promote participation and accountability that is deemed ethical.” 
```

Comment:
it doesn't like this `[T]` because that usually means it's a link. You need to escape the brackets with back slashes, like: `\[T\]`